docs/cluster_up_down.md

```diff
@@ -320,6 +321,26 @@ You can install logging aggregation components by specifying the --logging argum
 With logging aggregation installed, a new link will appear in the logs tab of a running pod in the web console.
 
 
+## Installing the Service Catalog
+
+| NOTE |
+| ---- |
+| This feature requires an oc command v3.6 or newer |
+
+You can enable the service catalog component by specifying the --service-catalog argument when invoking `oc cluster up`.
+
+Enabling the service catalog has the following effect:
+
+1) The api aggregator is enabled to provide a unified api endpoint for both OpenShift/Kubernetes resources and the new apis introduced by the service catalog.
```

Comment:
- API (all-caps).
- "1) The api aggregator ..." can be formatted with ". The api aggregator ..." if in asciidoc.